Documentation-rendertest/Directives/youtube.rst

```diff
@@ -1,11 +1,11 @@
 
 .. include:: /Includes.rst.txt
 
-.. _sphinxcontrib-youtube:
+.. _youtube-directive:
 
-=====================
-sphinxcontrib-youtube
-=====================
+=================
+Youtube directive
+=================
 
 See https://github.com/TYPO3-Documentation/sphinx-contrib-youtube/blob/develop/README.rst
```

Comment:
I assume, the link should be removed?

Also further below the sphinxcontrib.youtube README part?

Reply:
Thanks for catching this, Done